acquia_cms.profile

```diff
@@ -88,16 +90,54 @@ function acquia_cms_install_tasks(): array {
  * Send heartbeat event after site installation.
  */
 function acquia_cms_send_heartbeat_event() {
+  $cohesion_configured = $config->get('api_key') && $config->get('organization_key');
+  // Get the install & rebuild time from state API.
+  $install_start_time = \Drupal::state()->get('install_start_time');
+  $install_end_time = \Drupal::state()->get('install_end_time');
+  $rebuild_start_time = \Drupal::state()->get('rebuild_start_time');
+  $rebuild_end_time = \Drupal::state()->get('rebuild_end_time');
+  // Calculate the time diff from the function and pass it to telemetry.
+  $install_time_diff = acquia_cms_calculate_time_diff($install_start_time, $install_end_time);
+  $rebuild_time_diff = acquia_cms_calculate_time_diff($rebuild_start_time, $rebuild_end_time);
   Drupal::configFactory()
     ->getEditable('acquia_telemetry.settings')
     ->set('api_key', 'e896d8a97a24013cee91e37a35bf7b0b')
     ->save();
   \Drupal::service('acquia.telemetry')->sendTelemetry('acquia_cms_installed', [
     'Application UUID' => Environment::getAhApplicationUuid(),
     'Site Environment' => Environment::getAhEnv(),
+    'Install Timme' => $install_time_diff
+  ]);
+  \Drupal::service('acquia.telemetry')->sendTelemetry('acquia_cms_site_studio_rebuild_time', [
+    'Application UUID' => Environment::getAhApplicationUuid(),
+    'Site Environment' => Environment::getAhEnv(),
+    'Rebuild Time' => $rebuild_time_diff,
   ]);
```

Comment:
@panshulK We need to send one more variable to telemetry i.e `acquia_cms_site_studio_installed`.

Reply:
Should we send this explicitly?
Because I can see that this is already being set and send as part of the code : 
```
  \Drupal::service('acquia.telemetry')->sendTelemetry('acquia_cms_installed', [
    'Application UUID' => Environment::getAhApplicationUuid(),
    'Site Environment' => Environment::getAhEnv(),
    'Install Timme' => $install_time_diff,
  ]);
  ```
  I just added the install_time_diff variable here.

Reply:
@panshulK Yes we've to send this explicitly. With value true or false. 
True if site studio is installed, false if site studio is not installed.

<img width="902" alt="Screenshot 2021-06-23 at 8 42 58 PM" src="https://user-images.githubusercontent.com/81069167/123122409-99ac7880-d463-11eb-92d3-f215dc2ef09f.png">

Reply:
Done @vishalkhode1